scripts/prohibit-java-code-commented-out

```diff
@@ -5,6 +5,6 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
-if git --no-pager grep -En '^ *// ?(@.*|.*\{|.*\);)$' -- '*.java'; then
+if git --no-pager grep -En --perl-regexp '^ *// ?(@((?!formatter).)*|.*\{|.*\);)$' -- '*.java'; then
```

Comment:
You can take advantage of the fact that annotations in Java are virtually always upper-case:

```suggestion
if git --no-pager grep -En '^ *// ?(@[A-Z].*|.*\{|.*\);)$' -- '*.java'; then
```

Reply:
(also, if you anyway stick to `-P`/`--perl-regexp`, then `-E`/`--extended-regexp` is redundant (`-P` is practically a superset of `-E`)

Reply:
I thin it's more clear when it is as is. It indicates what I exactly want to exclude from this check.